docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1857cc7",
+   "metadata": {},
+   "source": [
+    "Before training our machine learning model, we must first obtain the consensus labels from the annotators that labeled the data. The simplest way to obtain an initial set of consensus labels is to select it using majority vote."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d009f347",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "majority_vote_label = get_majority_vote_label(multiannotator_labels)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7287b733",
+   "metadata": {},
+   "source": [
+    "Next, we will train our model on the consensus labels obtained using majority vote to compute out-of-sample predicted probabilities. Here, we use a simple logistic regression model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cbd1e415",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.linear_model import LogisticRegression\n",
+    "from sklearn.model_selection import cross_val_predict\n",
+    "\n",
+    "model = LogisticRegression()\n",
+    "\n",
+    "num_crossval_folds = 5  \n",
+    "cv_pred_probs = cross_val_predict(\n",
+    "    estimator=model, X=X, y=majority_vote_label, cv=num_crossval_folds, method=\"predict_proba\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4eab5188",
+   "metadata": {},
+   "source": [
+    "## 4. Use cleanlab to get better consensus labels and other statistics"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d392ce5",
+   "metadata": {},
+   "source": [
+    "Using the annotators' labels and the out-of-sample predicted probabilites from the model, cleanlab can help us obtain improved consensus labels for our data.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6ca92617",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "multiannotator_dict = get_label_quality_multiannotator(multiannotator_labels, cv_pred_probs)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "98042e7f",
+   "metadata": {},
+   "source": [
+    "Here, we use the `multiannotator.get_label_quality_multiannotator()` function which returns a dictionary containing three items:\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76d7c0e2",
+   "metadata": {},
+   "source": [
+    "- `label_quality_multiannotator` which gives us the improved consensus labels using information from each of the annotators and the model. The DataFrame also contains information about the number of annotations, annotator agreement and consensus quality score for each example.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf945113",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"label_quality_multiannotator\"].head()"
```

Comment:
maybe let's reorder the columns more thematically so the column order is: 
consensus_label, consensus_quality_score, annotator_agreement, num_annotations

What do you think?

Reply:
Let's also shorten the string name of this dict key to just be "label_quality" instead of "label_quality_multiannotator" ?

I presume it's clear to user that everything in this module is related to multiannotator

Reply:
I originally arranged `num_annotations` to be the first column because that would be the only non-repeating column if we passed more than one `consensus_method` (in a list). But I think the order you proposed makes more sense since `consensus_label` is probably the most important column